### PR TITLE
Grammar and Syntax fixes for April posts

### PR DESCRIPTION
**Pull request by amoreauCoveo**
Some small fixes around language. Mostly syntax, but some word changes as well. When I did change a word, I tried to keep the original tone as much as possible :)

### CODE REVIEW

File: _posts/2023-04-11-temporary-privileges.md [hunk 1]
```diff
@@ -121,17 +121,17 @@ For those of you who spent time on the first YAML block of this post, you notice
...
+Inside our AWS accounts, the Coveo R&D has [tag policies](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_tag-policies.html) that tell our engineers to set some specific tags with a given set of allowed values. When code registers an instance or a database inside strongDM, the code gives strongDM those tags, which then allows us to filter resources in strongDM based on them. From there, R&D engineers that want to contribute to the access matrix can write rules, like "a `platform` team engineer with the `production_administrator` access level who can request an `ssh` access to instances of the `prod` environment where the instance `team` tag value matches `platform`, if all the requirements are met".
```
Comment (JPLachance) [inline _posts/2023-04-11-temporary-privileges.md:124]: Here, "access level who can request an `ssh` access to instances", my goal was to put in words and "access rule".

Therefore, the formulation that has the right meaning in my mind is the original one.

"access level can request an `ssh` access to instances"

**Review by JPLachance — APPROVED** [review]
Thank you!

File: _posts/2023-04-11-temporary-privileges.md (hunk 1 above)
Comment (amoreauCoveo) [reply _posts/2023-04-11-temporary-privileges.md:124]: Ahhh sorry I think I just got your mearning. There's a lot of accesses in that sentence and I think I got confused

**Review by klalumiere — APPROVED** [review]
Thanks! ❤️

**Pull request merged**